mix.exs

```diff
@@ -107,6 +108,15 @@ defmodule ExDoc.Mixfile do
   defp test_dev_examples(:dev), do: Path.wildcard("test/examples/*")
   defp test_dev_examples(_), do: []
 
+  defp redirects(:dev) do
+    %{
+      "old-admonition" => "admonition",
+      Exdoc.OldMarkdown => ExDoc.Markdown
+    }
+  end
+
+  defp redirects(_), do: %{}
+
```

Comment:
We should not add those here and instead have a proper test. :)

Reply:
Oops 🤦

Reply:
Test added, removed from here.